apps/apps.go

```diff
@@ -19,9 +19,10 @@ const workflowURLPrefix = "deis."
 // ErrNoLogs is returned when logs are missing from an app.
 var ErrNoLogs = errors.New(
 	`There are currently no log messages. Please check the following things:
-1) Logger and fluentd pods are running.
+1) Logger and fluentd pods are running: kubectl get pods --namespace=deis.
```

Comment:
Nitpick: if you always do the namespace flag first, it's easier to make a shell alias IMHO:
```
alias kd='kubectl --namespace=deis'
```